criu/include/cr_options.h

```diff
@@ -63,6 +63,20 @@ struct cg_root_opt {
 
 #define DEFAULT_TIMEOUT		10
 
+/*
+ * File validation options
+ */
+#define FILE_VALIDATION_FILE_SIZE		1
+#define FILE_VALIDATION_BUILD_ID		2
+#define FILE_VALIDATION_CHKSM			3
+#define FILE_VALIDATION_CHKSM_EVERY		4
```

Comment:
Need to add a comment which describes what each of these constant means.

And I think they have to be in images/regfile.proto as enum

Reply:
Why does the enum need to be in images/regfile.proto? Currently, I've left it in criu/include/cr_options.h.